mobile/src/main/java/org/openhab/habdroid/ui/AboutFragment.java

```diff
@@ -10,36 +10,45 @@
 package org.openhab.habdroid.ui;
 
 
-import android.app.Dialog;
 import android.os.Bundle;
-import android.support.v4.app.DialogFragment;
+import android.support.v4.app.Fragment;
 import android.text.Html;
 import android.text.method.LinkMovementMethod;
 import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import org.openhab.habdroid.R;
 import android.widget.TextView;
 
+import org.openhab.habdroid.BuildConfig;
+import org.openhab.habdroid.R;
+
+import java.text.DateFormat;
 import java.text.SimpleDateFormat;
 import java.util.Calendar;
+import java.util.Locale;
 
-public class AboutFragment extends DialogFragment {
+public class AboutFragment extends Fragment {
 
     private static final String TAG = AboutFragment.class.getSimpleName();
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
 
-        View view = inflater.inflate(R.layout.openhababout, container);
+        View view = inflater.inflate(R.layout.openhababout, container, false);
 
-        TextView about = (TextView)view.findViewById(R.id.license_link);
+        TextView about = (TextView) view.findViewById(R.id.license_link);
         about.setText(Html.fromHtml(getString(R.string.about_license)));
         about.setMovementMethod(LinkMovementMethod.getInstance());
+        TextView appVersion = (TextView) view.findViewById(R.id.app_version);
+        appVersion.setText(getString(R.string.about_version_string,
+                BuildConfig.VERSION_NAME,
+                DateFormat.getDateTimeInstance().format(BuildConfig.buildTime)
+        ));
 
-        String year = new SimpleDateFormat("yyyy").format(Calendar.getInstance().getTime());
+        String year = new SimpleDateFormat("yyyy", Locale.US)
```

Comment:
Why locale US?

Reply:
Uh, I wasn't even sure that I've commited this change :) My IDE (IntelliJ) suggested to use an explicit Locale over SimpleDateFormat directly. As this is only a year, I though Locale.US fits in all cases.

Reply:
Also in chinese? I think we can ignore China as long there are no chinese strings.

Reply:
Good argument, as this is nothing, which is required in this commit, I'll revert this particular change :)

Reply:
I thought about this and now I think the change to Locale.US is usefull. Imagine some country with other years, than they would see "Copyright 2010 - 84512" or something like that.